doc/README.md

```diff
@@ -0,0 +1,53 @@
+List of documents
+=================
+
+|Folder         | Document Name     |
+|---------------|-------------------|
+| .             | SDK documentation |
+| user_guide    | User Guide        |
+| release_notes | Release Notes     |
+
+As of now you're supposed to build all the documents separately.
+
+Tested with Ubuntu 24.04. The goal is to build the documentation in html- and pdf-format.
```

Comment:
@VeijoPesonen standard Ubuntu support for zas is 22.04.

Reply:
Removed the sentence.